vortex-btrblocks/benches/compress.rs

```diff
@@ -1,5 +1,8 @@
 #![allow(clippy::unwrap_used)]
+#![allow(unexpected_cfgs)]
+#![allow(unused_imports)]
```

Comment:
I'll think about a cleaner design here, but want to get `develop` back to green again for now.